packages/manager/modules/pci/src/projects/project/serving/namespace/infos/infos.html

```diff
@@ -0,0 +1,78 @@
+        <oui-tile class="h-100" data-heading="{{:: 'pci_projects_project_serving_namespace_infos_namespace_information' | translate }}">
+            <oui-tile-definition data-term="{{:: 'pci_projects_project_serving_namespace_infos_namespace_description' | translate }}">
+                <oui-tile-description data-ng-bind="$ctrl.namespace.description"></oui-tile-description>
+            </oui-tile-definition>
+            <oui-tile-definition data-term="{{:: 'pci_projects_project_serving_namespace_infos_namespace_region' | translate }}">
+                <oui-tile-description data-ng-bind="$ctrl.namespace.region"></oui-tile-description>
+            </oui-tile-definition>
+            <div class="oui-tile__item">
+                <dl class="oui-tile__definition">
+                    <dt class="oui-tile__term">
+                        <span data-translate="pci_projects_project_serving_namespace_infos_namespace_container"></span>
+                    </dt>
+                    <dd class="oui-tile__description d-flex align-items-left" data-ng-click="$ctrl.goToContainer()">
+                        {{ $ctrl.namespace.container }}
+                    </dd>
+                </dl>
+            </div>
+
+            <oui-tile-definition data-ng-if="$ctrl.registry.custom" data-term="{{:: 'pci_projects_project_serving_namespace_infos_namespace_registry' | translate }}">
+                <oui-tile-description data-ng-bind="$ctrl.registry.registry"></oui-tile-description>
+            </oui-tile-definition>
+            <oui-tile-definition data-ng-if="!$ctrl.registry.custom"
+                                 data-term="{{:: 'pci_projects_project_serving_namespace_infos_namespace_container' | translate }}"
+                                 data-description="{{:: 'pci_projects_project_serving_namespace_infos_namespace_registry_shared' | translate }}">
+            </oui-tile-definition>
+
+        </oui-tile>
+    </div>
+    <div class="col-md-4">
+        <oui-tile data-heading="{{ ::'pci_projects_project_serving_namespace_infos_namespace_information_service' | translate }}">
+            <div class="oui-tile__item">
+                <dl class="oui-tile__definition">
+                    <dt class="oui-tile__term">
+                        <span data-translate="pci_projects_project_serving_namespace_infos_hub_url"></span>
+                        <oui-popover data-oui-popover-placement="auto">
+                            <button type="button" class="oui-popover-button" data-oui-popover-trigger></button>
+                            <oui-popover-content>
+                                <span data-translate="pci_projects_project_serving_namespace_infos_hub_url_help"></span>
+                            </oui-popover-content>
+                        </oui-popover>
+                    </dt>
+                    <oui-tile-description>
+                        <oui-clipboard model="'https://hub.c1.serving.ai.ovh.net'"></oui-clipboard>
```

Comment:
This should be extracted to a constant file. Moreover isn't it supposed to be returned by API ?

Reply:
fixed 9eb0962dad0527f7e808bd6b11ebe3c3cfec136e